### DISCUSSION TITLE
Change Difficulty?

### DISCUSSION THREAD

**Discussion by JulianHHS** [KillahPotatoes/KP-Liberation#881] [Q&A]
Hi, whenever I sett my difficulty to veteran in my TADST settings it doesn't actually change the parameters? is this normal? Does it change in the background and just doesnt show? Or does liberation overwrite everything that's been set in the server config?

I need to find a way to be able to play on the "easy/recruit" difficulty while having reduced damage off, because when i play on Lythiumm, it says having reduced damage on could cause issues..

any help would be appreciated!

**Comment by veteran29** [comment]
Mission parameters are unrelated to server difficulty, the one you set in the parameter is the difficulty of the mission itself (amount of enemies, armour usage etc.).

**Comment by Mach2Infinity** [comment]
@JulianQwerty in TADST under the Missions tab ensure you've selected **custom** from the difficulty drop-down menu and clicked the **green tick** to apply the change. Then under the Difficulty tab make sure Reduced Damage is **unticked**. See the screenshot below:

![image](https://user-images.githubusercontent.com/64335855/123562192-047feb80-d7a5-11eb-805b-08dc702d9447.png)

**Reply by Baker31b** [reply]
Is there a reason when i change the Difficulty like stated above in my TADST server it changes it back when i hit save? its not running or anything.
